tests/odb/mixed.c

```diff
@@ -261,5 +261,20 @@ void test_odb_mixed__expand_ids(void)
 	cl_git_pass(git_odb_expand_ids(_odb, ids, num));
 	assert_notfound_objects(ids);
 	git__free(ids);
-}
 
+	/* test looking for the actual (correct) types after accessing the object */
```

Comment:
I'd argue that all the previous tests should be split into their own functions. While it's fine to leave existing code as-is, I'd prefer to at least do the right thing with this new test.

Reply:
Done